admin_guide/out_of_resource_handling.adoc

```diff
@@ -47,10 +48,13 @@ description column based on the node summary API.
 |`memory.available` = `node.status.capacity[memory]` - `node.stats.memory.workingSet`
 |===
 
+[NOTE]
+====
 In future releases, the node will support the ability to trigger eviction
 decisions based on disk pressure. Until then, use
 xref:../admin_guide/garbage_collection.adoc#admin-guide-garbage-collection[container
 and image garbage collection].
+====
 
 [[out-of-resource-eviction-thresholds]]
 === Eviction Thresholds
```

Comment:
I think what's missing is an explanation of what an eviction threshold, and why you would want to use a soft one over a hard one.